index.bs

```diff
@@ -291,10 +291,13 @@ A trigger state is a [=struct=] with the following items:
 
 A randomized source response is null or a [=set=] of [=trigger states=].
 
+<h3 dfn-type=dfn>Attribution filter value</h3>
```

Comment:
Could we drop the attribution name from these types as well?

We could maybe make this a bit more concises by doing:

<h3>Attribution filtering</h3>
A <dfn>filter value</dfn> is an [=ordered set=] of [=strings=].

A <dfn>filter map</dfn> is an [=ordered map=] whose [=map/key|keys=] are [=strings=] and whose
[=map/value|values=] are [=filter values=].